addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -19,153 +17,241 @@
 
 <template id="index" name="Find Resellers">
     <t t-call="website_crm_partner_assign.layout">
-        <div class="col-lg-12">
-            <h1 class="text-center">
-                Looking For a Local Store?
-            </h1><h2 class="text-center text-muted">
-                Contact a reseller
-            </h2>
-        </div>
-
-        <div class="col-lg-3 mb32" id="partner_left">
-
-            <ul id="reseller_grades" class="nav nav-pills flex-column mt16">
-                <li class="nav-header nav-item"><h3>Filter by Level</h3></li>
-                <t t-foreach="grades" t-as="grade">
-                    <li class="nav-item">
+        <div class="o_wcrm_filters_top d-flex justify-content-end flex-wrap gap-2 w-100 d-print-none">
+        <!-- Is there a way to retrieve the active country without looping ? -->
+            <t t-foreach="countries" t-as="country">
+                <h4 t-if="country['active']" class="my-0 me-auto pe-sm-4 lh-base">Find a reseller <span t-if="country['country_id'][0] != 0">in <t t-out="country['country_id'][1]"></t></span></h4>
```

Comment:
`lh-base` : it's a fortunate coincidence that it matches the toolbar's height. If we want to vertically align title and toolbar I'd prefer to do it explicitly by using `align-items-center`